examples/disaster-recovery/README.md

```diff
@@ -0,0 +1,50 @@
+# Coordinate disaster recovery with Secret Manager
+
+## Background
+
+The Auth Proxy doesn't currently have any support for automatically
+following a [switchover operation][switchover].
+
+This document shows a workaround where you wrap the Auth Proxy in a
+script that polls a secret in Secret Manager. The secret holds the
+instance URI of your active primary instance. When that secret changes
```

Comment:
How would the secret change to become the new primary instance? Something like a separate script to update the secret whenever switchover is performed?

Reply:
Ah okay. Guess that's better than nothing